content/articles/django-custom-template-tags-and-filters/index.md

```diff
@@ -0,0 +1,224 @@
+
+### Introduction.
+We are going to cover how to create Django custom template tags and filters.Django template language comes with built-in template tags such as `{% if %}` and filters such as `safe`.You may however find that  you require a functionality not included in django.You create them using python and avail them using the `load ` tag.Let’s get started! 
+###  Prerequisites.
+For you  to follow along its important that:
+1. You are able to create a simple django application
+2. You have python and django installed.
+3. An suitable IDE such as Pycharm, VS Code etc.
+### Takeaways.
+1. Understand  use of custom template tags and filters and how they work.
+2. Create your own customized template tags and filters to suit your needs.
+### Getting started.
+Your custom tags and filters must be within a django app. If you wish that your tags and filters be used across your entire project, you may create an app just to accomplish that.
+
+Create an app called Contacts and add it in your INSTALLED_APPS.Inside your app , create a directory called templatetags in the same level as views,  models etc.
+Make the templatetags directory a package in python by creating an `__init__.py` file.Create another file custom_tags.py or any other name that will hold your custom tags
+and filters.
```

Comment:
Convert this into an ordered list instead of paragraphs.

Reply:
Done :rocket: